### PR TITLE
Feature/cldn 1282

### PR DESCRIPTION
**Pull request by cccs-Dustin**
### SUMMARY

This is the PR for the associated CLDN-1282 task

Changes include adding a new endpoint to the back-end that serves as a proxy to Alfred, as well as the modification of the Application Links viz to work with this new endpoint. The control panel of the Application Links viz was also modified so that the chart will display properly regardless of the dataset that it is made on.

It is not seen in this PR, but I also had to edit the extra-ca-certs Kubernetes secret on the PB-stg cluster by adding in a .pem file from the Alfred website. 

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

N/A

### TESTING INSTRUCTIONS

N/A

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [X] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Review by dparent1 — CHANGES_REQUESTED** [review]
I'd like to see an __init__ method in the class unless there is a strong reason to avoid one.

File: superset/proxy/api.py [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+    datamodel = SQLAInterface(SqlaTable)
+
+    include_route_methods = {"get_userid", "get_ipstring"}
+    resource_name = "proxy"
+
+    openapi_spec_tag = "Proxy"
+
+    user = current_user
+
+    ALFRED_SCOPE = os.environ.get("ALFRED_SCOPE")
```
[the remark this answers is not in the document]
Comment (cccs-Dustin) [reply superset/proxy/api.py:39]: I can definitely do that, the __init__ method will look like:

`def __init__(self):
        """
        This is the init function for the ProxyRestAPI class
        """
        super().__init__()
        self.user = current_user

        self.ALFRED_SCOPE = os.environ.get("ALFRED_SCOPE")
        if self.ALFRED_SCOPE is None:
            self.ALFRED_SCOPE = "api://alfred.pb/Alfred.ALL"

        self.ALFRED_URL = os.environ.get("ALFRED_URL")
        if self.ALFRED_URL is None:
            self.ALFRED_URL = "https://alfred-stg-pb.chimera.cyber.gc.ca"

        self.SSL_CERT = os.environ.get("REQUESTS_CA_BUNDLE")`

**Review by dparent1 — APPROVED** [review]

**Comment by cccs-Dustin** [comment]
Closing this PR as I will open another one on the cccs-1.5 branch

**Comment by cccs-Dustin** [comment]
Closing this PR as I will open another one on the cccs-1.5 branch

**Pull request closed without merging**